server/journeys/record-investigation/interviews-summary/view.njk

```diff
@@ -0,0 +1,118 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set fullWidth = interviews.length %}
+{% set pageTitle = "Interviews summary - Record a CSIP investigation " %}
+
+{% block innerContent %}
+    <span class="govuk-caption-l">Record a CSIP investigation</span>
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Interviews summary</h1>
+
+    {% if interviews.length %}
```

Comment:
Do we need this if? In most languages if you do a for-in or for-each loop, if there's nothing in it it'll just exit straight away.